test/system/drag_and_drop_test.rb

```diff
@@ -1,29 +1,29 @@
-require 'application_system_test_case'
+# require 'application_system_test_case'
```

Comment:
I'd suggest throwing a comment at the top of this file about the misbehavior, so we can fix it at a later date without reinvestigating.